galasa-inttests-parent/dev.galasa.inttests/src/main/java/dev/galasa/inttests/docker/AbstractDockerUbuntuLocal.java

```diff
@@ -99,16 +99,22 @@ private void installDocker() throws Exception {
 		storeOutput("ecosystem/dockerSetup", "startDocker.txt", "Command: sudo systemctl start docker \nOutput:\n" + res);
 		res = shell.issueCommand("sudo systemctl show --property ActiveState docker");	
 		Instant expire = Instant.now().plus(3, ChronoUnit.MINUTES);
+		Instant trace = Instant.now().plus(1, ChronoUnit.MINUTES);
```

Comment:
instead of "trace",   how about "statusReport",      makes it clear what it is for